suite-native/qr-code/src/components/QRCode.tsx

```diff
@@ -32,6 +32,25 @@ export const QRCode = ({ data }: QRCodeProps) => {
     const { applyStyle } = useNativeStyles();
     const [originalBrightnessValue, setOriginalBrightnessValue] = useState<number>();
 
+    useEffect(() => {
+        const subscription = AppState.addEventListener('change', nextAppState => {
+            if (!originalBrightnessValue) return;
+
+            // When app goes to background, restore the original brightness value.
+            if (nextAppState === 'inactive') {
```

Comment:
```suggestion
            if (nextAppState === 'background') {
```
The `inactive` state is supported only by iOS. More here: https://reactnative.dev/docs/appstate#app-states.

Reply:
good point, thnks

Reply:
cfb332afa